setup.py

```diff
@@ -54,23 +54,26 @@
         'jsonschema>=3.2.0',
         'jupyter_core>=4.0,<5.0',
         'jupyter_client>=6.1',
-        'jupyterlab-git==0.23.1',
-        'jupyterlab>=2.0.0,<3.0.0',
+        'jupyter_server>=1.0.5',
+        'jupyterlab>==3.0.0rc9',
```

Comment:
should we be using rc10?
```suggestion
        'jupyterlab>==3.0.0rc10',
```

Reply:
if you've tested that things still work on rc10 feel free to commit this, I'm in the middle of some coding and can't check out and test it rn

Reply:
I haven't tested out but I can help looking into that

Reply:
I have been using rc10, will commit it